cmd/litcli/ln.go

```diff
@@ -322,23 +413,15 @@ func sendPayment(cliCtx *cli.Context) error {
 	defer cleanup()
 
 	switch {
-	case !cliCtx.IsSet(assetIDFlag.Name):
-		return fmt.Errorf("the --asset_id flag must be set")
 	case !cliCtx.IsSet("keysend"):
 		return fmt.Errorf("the --keysend flag must be set")
 	case !cliCtx.IsSet(assetAmountFlag.Name):
 		return fmt.Errorf("--asset_amount must be set")
 	}
 
-	assetIDStr := cliCtx.String(assetIDFlag.Name)
-	assetIDBytes, err := hex.DecodeString(assetIDStr)
+	assetIDBytes, groupKeyBytes, err := parseAssetIdentifier(cliCtx)
 	if err != nil {
-		return fmt.Errorf("unable to decode assetID: %v", err)
-	}
-
-	assetAmountToSend := cliCtx.Uint64(assetAmountFlag.Name)
-	if assetAmountToSend == 0 {
```

Comment:
We don't need to check for zero amounts? Or is this offloaded to tapd?

Reply:
Hmm, you're right. The removal of this was left over from a version where I thought we'd also want the `sendpayment` command to work with invoices, which I've rolled back in the meantime (see https://github.com/lightninglabs/lightning-terminal/pull/1052#issuecomment-2858052649). So this check is still needed, added it back.